tsconfig.json

```diff
@@ -11,10 +11,12 @@
     "strict": true,
     "resolveJsonModule": true,
     "esModuleInterop": true,
+    "allowJs": true,
+    "declaration": true,
```

Comment:
Do we need `declarationMap` as well? I always get a little confused on TypeScript compiler settings.

(If we don't, please explain to me why not).

Reply:
declarationMap just adds the support for mapping declaration files to their original source. When distributing the build we don't include the original `src` directory so having the maps here would be pointless unless we included the original source files in the build.